apps/ledger-live-mobile/src/hooks/useSystem/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { useWindowDimensions, Platform } from "react-native";
+import { useSelector } from "react-redux";
+import { languageSelector, localeSelector, themeSelector } from "~/reducers/settings";
+
+const useSystem = () => {
+  // Device dimensions
+  const { width, height } = useWindowDimensions();
+
+  // Device informations
+  const os = Platform.OS;
+  const version = Platform.Version;
+
+  // Language informations
+  const language = useSelector(languageSelector);
+  const locale = useSelector(localeSelector);
+
+  // Styles
+  const theme = useSelector(themeSelector);
```

Comment:
I think that we should not rely on the store here. 
`language` and `locale` are not system information because they come from the settings and can be changed by the user, the same for the `theme`.
What I would expect from a `useSystem` hook would be system information (default language of the device, dark/light/auto mode) and not overridden information made by the user in the app.

Reply:
Sounds good to me.